tests/cases/fourslash/getJavaScriptCompletions16.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference path="fourslash.ts" />
+
+// @allowNonTsExtensions: true
+// @Filename: file.js
+//// "use strict";
+//// 
+//// class Something {
+//// 
+////     /**
+////      * @param {number} a
+////      */
+////     constructor(a, b) {
+////         a./**/
+////     }
+//// }
+
+goTo.marker();
+verify.completionListContains('toFixed', undefined, undefined, 'method');

```

Comment:
Could we also verify that at a `new` site you get the correct intellisense/signature help? (i.e. `var x = new Something(/* here */)`)